src/components/layout/layout.tsx

```diff
@@ -44,7 +46,8 @@ const Layout: React.FC<LayoutProps> = (props) => {
       <GlobalStyle isIndexPage={props.isIndexPage} />
       <Header
         siteTitle={data.site.siteMetadata.title}
-        light={props.isIndexPage}
+        light={props.isIndexPage || Boolean(props.heroImage)}
```

Comment:
light is a number and is given a boolean 🤔

Reply:
See answer above: Number only needed for styled components.